<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TransactionView</class>
 <widget class="QWidget" name="TransactionView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>652</width>
    <height>391</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Transactions</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Customer:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cboCustomers">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Start Date:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="dtpStartDate">
       <property name="date">
        <date>
         <year>2013</year>
         <month>2</month>
         <day>11</day>
        </date>
       </property>
       <property name="displayFormat">
        <string>dd-MMM-yyyy</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>End Date:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="dtpEndDate">
       <property name="date">
        <date>
         <year>2013</year>
         <month>2</month>
         <day>11</day>
        </date>
       </property>
       <property name="displayFormat">
        <string>dd-MMM-yyyy</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Transaction Type:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cboTxnType">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <item>
        <property name="text">
         <string>All</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Invoice</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Payment</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Credit Note</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>General Journal</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeView" name="trvData">
     <property name="alternatingRowColors">
      <bool>false</bool>
     </property>
     <attribute name="headerStretchLastSection">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
